nameserver/dns.go

```diff
@@ -92,3 +96,18 @@ func getMaxReplyLen(r *dns.Msg, proto dnsProtocol) int {
 	}
 	return maxLen
 }
+
+// shuffleAnswers reorders answers for very basic load balancing
+func shuffleAnswers(answers []dns.RR) []dns.RR {
+	n := len(answers)
+	if n > 1 {
+		rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Fixed. Added a init() function as @awh suggested, and changed the algorithm to the one provided by @rade.